Buddies/Activities/View Activity/ViewActivityController.swift

```diff
@@ -37,32 +37,22 @@ class ViewActivityController: UIViewController {
     
     /// Required for the `MessageInputBar` to be visible
     override var canBecomeFirstResponder: Bool {
-        
         if curMemberStatus! == .none {
```

Comment:
Got a fatal error on this line. `curMemberStatus` was `nil`. You should probably have a default that assumes `.none`.

Reply:
@JakeThurman I actually fixed this in some of my work. If we want that fix here:
```
let status = curMemberStatus ?? .none
if status == . none {}
```

Reply:
Fixed in the latest commit.